### ISSUE TITLE
JCenter is shutting down

### ISSUE THREAD

**Issue by kzsolti** [jfrog/artifactory-client-java#314]
https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

Are there any plans to publish to another repository now that JCenter is shutting down?

**Comment by harry-g** [comment]
I was also wondering where to find it in future.
Looking at the source code of the Jenkins Artifactory plugin, I saw it can be taken from this location:
https://oss.jfrog.org/artifactory/jfrog-dependencies/org/jfrog/artifactory/client/artifactory-java-client-services/
But is this an official/recommended location?
Documentation and examples should be updated to point to a valid replacement of jcenter.

**Comment by yahavi** [comment]
@kzsolti @harry-g 
During the Artifactory Java client release, we first publish the JARs to https://oss.jfrog.org. Until recently we also published the JARs to JCenter. Now we are working on publishing the JARs to Maven Central in addition to https://oss.jfrog.org. We'll update the documentation and examples as well. We'll keep you updated on that.

In case you encounter errors using JCenter, you can always use https://oss.jfrog.org as the source Artifactory:

**Gradle**
```gradle
repositories {
    maven {
        url "https://oss.jfrog.org/artifactory/jfrog-dependencies"
    }
}
```

**Maven**
```xml
<repository>
    <id>central</id>
    <name>jfrog-dependencies</name>
    <url>https://oss.jfrog.org/artifactory/jfrog-dependencies</url>
</repository>
```

**Comment by harry-g** [comment]
Hi @yahavi  , thanks a lot for the quick response.
So if oss.jfrog.org si safe to use, we will move to this one.

Or is it better to have both maven central (as first repo) and jcenter so it will kind of automatically switch over later?
In other words, is it guaranteed that you will put it to maven central _before_ jcenter is finally shut down?

**Comment by yahavi** [comment]
Hello @harry-g,
oss.jfrog.org is our main repository for external libraries and therefore it is safe to use.

The migration to Maven Central is currently under development. We'll make our efforts to release it in the next few days, before JCenter sunset. However, to be on the safe side, you can use oss.jfrog.org as the source Artifactory. We'll let you know once the JARs will be in Maven Central.

**Comment by yahavi** [comment]
@harry-g @kzsolti 
Artifactory Java client 2.9.2 is released and should be available now on Maven Central.
We'd appreciate your feedback on that.

**Comment by harry-g** [comment]
Looks fine and I see that the documentation is also updated!
Great, thanks a lot @yahavi !

**Comment by vdotjansen** [comment]
@yahavi the release of 2.10.0 did not make it to maven central. Maybe something to look into.

**Comment by yahavi** [comment]
@vdotjansen
Thanks for reporting this issue.
We published at this moment 2.10.0 to Maven Central. The version is expected to be indexed in Maven Central in a few hours.
